app/src/main/kotlin/com/hedvig/app/feature/claims/ui/commonclaim/CommonClaimActivity.kt

```diff
@@ -5,27 +5,26 @@ import android.content.Intent
 import android.net.Uri
 import android.os.Bundle
 import androidx.appcompat.app.AppCompatActivity
+import androidx.lifecycle.lifecycleScope
 import coil.ImageLoader
 import coil.load
 import com.hedvig.app.R
 import com.hedvig.app.databinding.ActivityCommonClaimBinding
 import com.hedvig.app.feature.claims.ui.commonclaim.bulletpoint.BulletPointsAdapter
-import com.hedvig.app.feature.claims.ui.pledge.HonestyPledgeBottomSheet
+import com.hedvig.app.feature.claims.ui.startClaimsFlow
 import com.hedvig.app.util.extensions.compatSetDecorFitsSystemWindows
-import com.hedvig.app.util.extensions.view.applyNavigationBarInsets
-import com.hedvig.app.util.extensions.view.applyStatusBarInsets
-import com.hedvig.app.util.extensions.view.disable
-import com.hedvig.app.util.extensions.view.enable
-import com.hedvig.app.util.extensions.view.setHapticClickListener
-import com.hedvig.app.util.extensions.view.setupToolbarScrollListener
+import com.hedvig.app.util.extensions.view.*
```

Comment:
ktlint will prob complain about this

It seems that now it failed here https://github.com/HedvigInsurance/android/actions/runs/3197099130/jobs/5219881747 with:
`The following Gradle properties are missing for 'odysseyRepository' credentials:`
so could it be that CI wasn't setup for it maybe, so it didn't have a chance to run